agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/AppIdSupplier.java

```diff
@@ -120,10 +123,9 @@ public void run() {
       HttpResponse response;
       try {
         response = LazyHttpClient.getInstance().send(request).block();
-      } catch (RuntimeException e) {
-        // TODO handle Friendly SSL exception
-        logger.debug(e.getMessage(), e);
-        backOff("exception sending request to " + url, e);
+      } catch (Throwable t) {
```

Comment:
is there particular reason for changing RuntimeException to Throwable here?

Reply:
in the legacy code we capture and log UnknownHostException and since UnknownHostException doesnot belong to Runtime Exception, I consolidated everything under Throwable.

Reply:
the try block doesn't throw `UnknownHostException`